src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,12 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    public String getPrintableString(Printable... printables) {
+        String toPrint = "";
+        for (Printable p : printables) {
```

Comment:
Try to use StringJoiner for string concatenation in a loop for better readability and performance.